docs/src/main/asciidoc/pubsub.adoc

```diff
@@ -25,6 +25,27 @@ dependencies {
 
 This starter is also available from https://start.spring.io[Spring Initializr] through the `GCP Messaging` entry.
 
+=== Spring Boot Actuator Support
```

Comment:
Why did you move this section up?
Our convention is to have configuration after deps in refdoc.

Reply:
I'll move it back. My thought was that configuration is really long for pubsub, and this needed to be more prominent, but maybe not.